database/rrdlabels.c

```diff
@@ -50,7 +54,8 @@
  *
  */
 
-#define LABELS_MAX_LENGTH 400 // 400 in bytes, up to 200 UTF-8 characters
+#define LABELS_MAX_NAME_LENGTH 200
+#define LABELS_MAX_VALUE_LENGTH 400 // 400 in bytes, up to 200 UTF-8 characters
```

Comment:
From this comment and later bellow we have a comment `// UTF-8 encoded 2-byte character` 

it seems we assume UTF-8 being 2 bytes per character. This is not true UTF-8 char can be 1-4 Bytes long

Reply:
fixed it